appkeeper/README.md

```diff
@@ -6,7 +6,22 @@ SIOS AppKeeper automatically restarts failed Amazon EC2 services when notificati
 
 ## Setup
 
-### Get the SIOS AppKeeper API key
+### Agent Installation
+
+The AppKeeper check is not included in the [Datadog Agent][14] package, so you need to install it yourself.
+See [the official community integration installation instructions][15].
+
+### Agent Configuration
+
+1. Edit the `appkeeper.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory.
+   Change the parameter values as follows:
+   * account: your AWS account
+   * integrationToken: The token you got in Setup step.1 above.
+
+2. [Restart the Agent][16].
+
+### AppKeeper Setting
+#### Step 1. Get the SIOS AppKeeper API Key
```

Comment:
The headings aren't necessary. I would write this section as nested ordered lists:
```
1. Get the SIOS AppKeeper API key from the AppKeeper GUI:
  1. Click **Account Information**, and open the modal dialog.
  2. Click **Get Token**.
  3. Copy the token.
2. Define the webhook in the Datadog integration dashboard:
...
```
and so on, all the way down to your current step 4. You could even work in the Agent Configuration set of steps (currently lines 14-21) into the list, in the appropriate place (somewhere after "Get token").